vars/runTest.groovy

```diff
@@ -99,6 +99,10 @@ def call(Map config = [:]) {
                           config['failure_artifacts'] + '"'
     }
 
+    if(config['with_valgrind']) {
```

Comment:
Is the key 'valgrind' or 'with_valgrind'?

Reply:
I'm using 'valgrind' for parseStageInfo and as the parameter for unitTest.groovy and 'with_valgrind' for setting up the environment variable, but I could stick to 'with_valgrind' for consistency